plugins/import_pil/oomox_plugin.py

```diff
@@ -185,7 +185,7 @@ class Plugin(OomoxImportPlugin):
 
     theme_model_import = [
         {
-            'display_name': _('Import colors from image: '),
+            'display_name': _('Import Colors from Image:'),
```

Comment:
Also should we live the colon at the end of the headline? HIG is not saying anything on this regard

Reply:
+1 for removing the colon at the end of the headline, but I didn't do that for this PR as I thought it is another issue.